fileszip/fileszip.go

```diff
@@ -1,4 +1,4 @@
-package coderd
+package fileszip
```

Comment:
Suggestion: archive, since we're dealing with both tar's and zip's.

Reply:
Renamed